tests/es/homeassistant_HassTurnOn.yaml

```diff
@@ -7,3 +7,9 @@ tests:
       name: "HassTurnOn"
       slots:
         name: "fan.ventilador_techo"
+  - sentences:
```

Comment:
While it's not incorrect,do we ever refer to the switch when turning on/off something? I always ask to turn on/off the light (or whatever) controlled by the switch, but never the switch itself. Perhaps there's an use case I'm not aware of.

Reply:
Sometimes you could have a generic switch in a room, where you connect several devices. 
In that case, you would use this sentence.